solr/bin/solr

```diff
@@ -1024,9 +912,12 @@ fi
 # so that we can write logs for examples to $SOLR_HOME/../logs
 : "${SOLR_LOGS_DIR:="$SOLR_SERVER_DIR/logs"}"
 EXAMPLE_DIR="$SOLR_TIP/example"
+
+# Set default log4j configuration
+LOG4J_PROPS="$DEFAULT_SERVER_DIR/resources/log4j2.xml"
```

Comment:
I guess this is still one of those "chicken/egg" things that you can't configure in the bootstrap?

Reply:
Yes, guess it needs to be set fairly early, but who knows, could probably be set programatically in main method as well...